frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -158,66 +158,66 @@ const en: Translations = {
           modified: 'Last modified'
         },
         attachmentBox: {
-          nb: 'Huom!',
+          nb: 'Please note!',
```

Comment:
Would just "Note!" be better as it uses a similar space to both "Huom!" and "Obs!"? Or is extra clarity preferable in this context?

Reply:
I dunno, this originated from the old vue translations. But I guess we could use the shorter form here.